### DISCUSSION TITLE
two machines two different nics

### DISCUSSION THREAD

**Discussion by noodlenoggan** [esnet/iperf#1421] [Q&A]
Hi, what would the commands be if I want to run iperf between the two PCs via their NIC2 interfaces? Thanks

PC1
NIC1 - 192.168.1.1
NIC2 - 12.12.12.1

PC2
NIC1 - 192.168.1.2
NIC2 - 12.12.12.2

**Comment by bmah888** [comment]
If you want PC2 to be the server, then on PC2 do `iperf3 --server`.

On PC1 (which will be the client) do `iperf3 --client 12.12.12.2` The argument to the `--client` option is the IP address of the server interface to connect to. Assuming you have NIC2 interfaces wired together directly or you have them on a common subnet, this should run all the test traffic over the NIC2 interfaces.
